.github/ISSUE_TEMPLATE/new_lint.md

```diff
@@ -0,0 +1,43 @@
+---
+name: New lint suggestion
+about: Suggest a new Clippy lint.
+labels: L-lint
+---
+
+### What it does
+
+*What does this lint do?*
+
+### Categories
+
+- Kind: *See <https://github.com/rust-lang/rust-clippy/blob/master/README.md#clippy> for list of lint kinds*
+
+*What benefit of this lint over old code?*
+
+For example:
+- Remove bounce checking inserted by ...
+- Remove the need to duplicating/storing/typo ...
+
+### Auto-apply suggestion ability
```

Comment:
This is also an internal Clippy thing, which can be derived from the example by the reviewer/person that implements it.